### PR TITLE
feat(Copy): update with new info

### PR DESCRIPTION
**Pull request by guilhermedecampo**

### CODE REVIEW

File: src/pages/join-us.js [hunk 1]
```diff
@@ -35,8 +35,13 @@ const CasesPage = () => (
           join us
         </Title>
         <Text marginBottom="50px">
-          We have a young team and that is part of the business. This brings motivation and focus in each project,
-          making it part of a collective effort, to exceed.
+          Over the years we have developed our own way to deal and help with each other. We like to enable dreams and
```
Comment (fakenickels) [inline src/pages/join-us.js]: not better to add this already in the i18n branch?

File: src/components/IntegrationServer.js [hunk 2]
```diff
@@ -48,9 +48,9 @@ const IntegrationServer = () => (
         <Lightbox cover={mobileIntegrationServer} images={[{ src: mobileIntegrationServer }]} />
       </MobileGameTableWrapper>
       <Text align="center">
-        Astrocoders builds <em>mobile and web apps</em>, <em>systems</em> and <em>platforms</em> in a different way.{' '}
-        <em>Simple</em>, <em>quick</em>, PCI & GDPR compliance and <em>made to last</em>. No unnecessary lines, no bends
-        or obstacles along the way.
+        The <em>integration server</em> enable us to connect with <em>any</em> provider and APIs to <em>process</em> and{' '}
+        <em>collocate</em> just the most <em>important data</em> into the apps we build or your own apps. We are able to
+        deploy it in your infrastructure or in our cloud based high-available servers.
```
Comment (fakenickels) [inline src/components/IntegrationServer.js]: 👍

File: src/pages/how-we-work.js [hunk 3]
```diff
@@ -160,15 +156,16 @@ const HowWeWork = () => (
...
+                build excellent financial systems. Within that in mind the ultimate goal is a long and healthy
+                partnership.
               </ProcessDescription>
             </Process>
             <Process>
               <ProcessTitle>Highly productive and focused team</ProcessTitle>
               <ProcessDescription className="content">
-                We have a young team and that is part of the business. This brings motivation and focus in each project,
-                making it part of a collective effort, to exceed expectations.
+                We have a top notch team that are up to any challenge in the financial world. We have always the
```
Comment (fakenickels) [inline src/pages/how-we-work.js]:
```suggestion
                We are a top notch team that are up to any challenge in the financial world. We have always the
```

File: src/pages/how-we-work.js [hunk 4]
```diff
@@ -160,15 +156,16 @@ const HowWeWork = () => (
             <Process>
               <ProcessTitle>We create relationships</ProcessTitle>
               <ProcessDescription className="content">
-                We accept projects only with customers that see value in the work and are really excited to built
-                excellent financial systems. Within that in mind the ultimate goal is a long and healthier partnership.
+                We accept only projects with engaged customers which see value in the work and are really excited to
```
Comment (fakenickels) [inline src/pages/how-we-work.js]:
```suggestion
                We accept only projects with engaged customers whom see value in the work and are really excited to
```
which or whom?

File: src/pages/how-we-work.js [hunk 5]
```diff
@@ -160,15 +156,16 @@ const HowWeWork = () => (
...
+                build excellent financial systems. Within that in mind the ultimate goal is a long and healthy
```
Comment (fakenickels) [inline src/pages/how-we-work.js]:
```suggestion
                build excellent financial systems. With that in mind the ultimate goal is a long and healthy
```

File: src/components/IntegrationServer.js (hunk 2 above)
Comment (eliabejr) [reply src/components/IntegrationServer.js]: I don't know if we'll be able to use a styled component inside a FormattedMessage

File: src/components/AboutUs.js [hunk 6]
```diff
@@ -55,17 +63,15 @@ const AboutUs = () => (
         </MobileGameTableWrapper>
 
         <Description>
-          State of the art tech and design which make your customer interface delightful and unbreakable
+          State of the art <Em>tech</Em> and <Em>design</Em> for <Em>delightful</Em> and <Em>unbreakable</Em> interfaces
```
Comment (eliabejr) [inline src/components/AboutUs.js]: I don't know if we'll be able to use a styled component inside a FormattedMessage

File: src/pages/join-us.js (hunk 1 above)
Comment (guilhermedecampo) [reply src/pages/join-us.js]: Yeah that branch didn't exist in the time I was working on it. 😞

**Review by eliabejr — APPROVED** [review]
I would only center Building Blocks and Join Us page descriptions, what do you think?

File: src/components/AboutUs.js (hunk 6 above)
Comment (guilhermedecampo) [reply src/components/AboutUs.js]: hmm yeah maybe you'll need to split over them.

Comment (eliabejr) [reply src/components/AboutUs.js]: would not better do exactly what we have below this component?

**Comment by guilhermedecampo** [comment]
Gonna do some final review on copy over here. Please don't merge yet =)

File: src/i18n/br.json [hunk 7]
```diff
@@ -1,43 +1,47 @@
...
+  "tableDescription": "<em>Tecnologia de ponta</em> e <em> design </em> para interfaces <em> delightful </em> e <em> sólidas</em>",
+  "whatOurCustomersSay": "O que nossos clientes dizem",
+  "quote1": "Astrocoders fornece software de alta qualidade e soluções para nossa plataforma de impostos.",
+  "quote2": "Astrocoders fez nosso fluxo personalizado de pagamentos funcionar com rapidez.",
+  "quote3": "Astrocoders foi essencial para descobrirmos como monetizar usando criptomoedas.",
+  "mobileSolutions": "Nós <em> usamos </em> e <em> criamos </em> tecnologia de ponta em nossos <em> componentes mobile financeiros</em>. Com centenas de possibilidades, você pode <em> personalizá-los </em> para suas necessidades. Com uma única base de código para aplicativos mobile nativos e nosso processo simplificado, podemos enviar novas versões para a <em> Apple Store </em> e <em> Google Play </em> automaticamente.",
```
Comment (fakenickels) [inline src/i18n/br.json]: "podemos personalizá-los"
I think sounds a little bit better so people don't think we are wix.
Make sense?

Comment (guilhermedecampo) [reply src/i18n/br.json]: yes makes sense!

**Review by fakenickels — APPROVED** [review]

**Pull request merged**